docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+services:
+  #  Create a service named db.
+  db:
+    image: "postgres:12-alpine"
+    container_name: "pipeline_db"
+    #   Setup the username, password, and database name
+    environment:
+      - POSTGRES_USER=pipeline
+      - POSTGRES_PASSWORD=pipeline
+      - POSTGRES_DB=pipeline
+    #   Map port 6432 (localhost) to port 5432 on the container
+    ports:
+      - "5432:5432"
```

Comment:
Why do we expose internal things to the outer world?
I suggest we make `db` and `gitlab` inaccessible from the host machines, create `network`, and add coresponding networks to the services:
```yml
version: "3"
services:
  db:
    # Other properties but no ports exposed
    networks:
      - db_network

  gitlab:
    # Other properties but no ports exposed
    networks:
      - git_network

  cromwell_pipeline:
    # Other properties
    ports:
      - 8080:8080
    networks:
      - db_network
      - git_network

networks:
  db_network:
    driver: overlay
  git_network:
    driver: overlay
```